content/src/content/jcr_root/apps/core/wcm/components/page/v3/page/page.html

```diff
@@ -22,6 +22,8 @@
     <head data-sly-call="${head.head @ page = page, pwa = pwa}"></head>
     <body class="${page.cssClassNames}"
           id="${page.id}"
+          data-cmp-link-accessibility-enabled
+          data-cmp-link-accessibility-text="${'opens in a new tab' @ i18n}"
```

Comment:
Would it be possible to expose a dictionary here with labels defined by the Model? This would allow us in CIF to reuse the approach without the need to overwrite the page.html? 

We could even introduce a new model interface "Dictionary" which is serialised to a json key-value map. Custom implementation could extend this using the delegation pattern and add any additional labels they need for the same/similar cases.

Reply:
the long-term solution is to find a replacement to the `Granite.i18n` clientlib so that we can internationalise strings on publish instances. Your suggestion would be a step towards this goal. IMHO this replacement should be part of the product. For now I would keep it as it is as a short term solution.

Reply:
Shall we have a ticket to track this long-term solution than?

Reply:
#1289 is about server side i18n. This PR is targeting to remove client side i18n based on the usage of Granite.I18n. For that it doesn't matter where the translations come from, but how they are exposed to a script on a site.

Reply:
#1289 is about replacing Sling Dictionaries with something maintainable at run time (no matter whether leveraging on server side with HTL i18n or on client side with some JS).